reference/orders.v2.oas2.yml

```diff
@@ -5244,6 +5301,11 @@ components:
           maxItems: 1
           items:
             $ref: '#/components/schemas/digitalConsignment_Put'
+        shipping:
+          type: array
+          minItems: 0
+          items:
+            $ref: '#/components/schemas/shippingConsignment_Put'
```

Comment:
maybe not in scope of this PR: in PUT request, we also support adding new consignment as well, so these should be "any of digitalConsignment_Put and digitalConsignment_Post", "any of shippingConsignment_Put and shippingConsignment_Post"